lib/elixir/lib/task.ex

```diff
@@ -526,15 +532,15 @@ defmodule Task do
   The shutdown method is either a timeout or `:brutal_kill`. In case
   of a `timeout`, a `:shutdown` exit signal is sent to the task process
   and if it does not exit within the timeout it is killed. With `:brutal_kill`
-  the task is killed straight away. In case the task exits abnormally, or a 
+  the task is killed straight away. In case the task exits abnormally, or a
   timeout shutdown kills the task, this function will exit with the same reason.
 
   It is not required to call this function when terminating the caller, unless
-  exiting with reason `:normal` or the task is trapping exits. If the caller is
-  exiting with a reason other than `:normal` and the task is not trapping exits the
+  exiting with reason `:normal` or if the task is trapping exits. If the caller is
+  exiting with a reason other than `:normal` and the task is not trapping exits, the
   caller's exit signal will stop the task. The caller can exit with reason
-  `:shutdown` to shutdown linked processes, such as tasks, that are not trapping
-  exits without generating any log messages.
+  `:shutdown` to shutdown all of its linked processes, including tasks, that
```

Comment:
I just wasn't sure whether it's correct to generalize the behaviour to "all linked processes". Didn't find any special treatment of the `shutdown` exit reason in the Erlang docs.